dashboard/app/util_test.go

```diff
@@ -89,6 +100,52 @@ func NewCtx(t *testing.T) *Ctx {
 	return c
 }
 
+var appIDSeq = uint32(0)
+
+func NewSpannerCtx(t *testing.T) *Ctx {
+	ddlStatements, err := loadDDLStatements("1_initialize.up.sql")
+	if err != nil {
+		t.Fatal(err)
+	}
+	// We need a unique AppID b/c spanner database is attached to the AppID.
+	// But don't use it if spanner is not used b/c it alters some outputs
+	// checked by existing tests.
+	appID := fmt.Sprintf("testapp-%v", atomic.AddUint32(&appIDSeq, 1))
+	uri := fmt.Sprintf("projects/%s/instances/%v/databases/%v", appID, aidb.Instance, aidb.Database)
+	spannertest.NewTestDB(t, uri, ddlStatements)
```

Comment:
I wonder how sustainable it is given that, over time, there will not be just `"1_initialize.up.sql"`. Why not drop the third parameter and just call `executeSpannerDDL` afterwards instead? Or some more generic `runMigrations()` that would load and then execute? That would simplify `CreateSpannerDB` changes.

Reply:
We can concatenate all of `*.up.sql` and pass as dll. Should work fine I think, and is faster (1 RPC instead of N+1 RPCs for each test).